src/vm/exceptionhandling.cpp

```diff
@@ -6184,11 +6187,11 @@ bool ExceptionTracker::IsInStackRegionUnwoundBySpecifiedException(CrawlFrame * p
 
     // Remember that sfLowerBound and sfUpperBound are in the "OS format".
     // Refer to the comment for CallerStackFrame for more information.
-#if defined(_TARGET_AMD64_)
+#ifndef USE_FUNCLET_CALL_HELPER
```

Comment:
@janvorli While digging this issue, I found that the major difference between AMD64 and ARM/ARM64 is that AMD64 uses the result of ``GetCurrentSP()`` called inside ``CallHandler`` to set ``m_EHClauseInfo``, but ARM/ARM64 updates it inside ``CallEHFunclet`` and ``CallEHFilterFunclet`` later (this is why I used ``USE_FUNCLET_CALL_HELPER`` here).

I agree with you that these things are not necessarily related, but then I would like to introduce another flag for these checks (for me, it was very difficult to find these checks are related when it was ifdefed by targets).

Could you let me know your opinion?